spec/acceptance/nodesets/ubuntu-16-docker.yml

```diff
@@ -0,0 +1,12 @@
+HOSTS:
+  ubuntu-16-docker:
+    roles:
+      - master
+    platform: ubuntu-1404-x86_64
```

Comment:
`FYI you're actually pulling 14.04, not 16.04 as Rakefile claims`

This was intentional.

`As discussed before, Beaker + Ubuntu 16.04 doesn't work out of the box. I tracked down the exact bug report for the issue we're hitting. Also, see this more generic add-Ubuntu 1604-support bug. I described the exact problem on the Beaker bug.`

Thanks a lot @dfarrell07. This is *very* helpful. I am working on it. Will send an updated PR soon. Thanks again.